File: python_ToSXs/python/06-thaphinh/96.py
height = 5

shape = ""
start = 1
end = 1

for i in range(1, height + 1):
    record = ""
    for j in range(start, end + 1):
        number = "0" + str(j) if j < 10 else str(j)
        record += number + " "

    shape += record + "\n"
    start = end + 1
    end = start + i

print(shape)

File: cpp_e9zMZ/cpp/06-thaphinh/96.cpp
#include <iostream>

#include "zvn.h"
using namespace std;

int main() {
    string shape = "";
    int height = 4;
    int start = 1;
    int end = 1;

    for (int i = 1; i <= height; i++) {
        string record = "";

        for (int j = start; j <= end; j++) {
            string number = j >= 10 ? to_string(j) : "0" + to_string(j);
            record += number + " ";
        }

        shape += record + "\n";
        start = end + 1;
        end = start + i;
    }
    
    cout << shape << "\n";
}